models/auth.py

```
@@ -0,0 +1,56 @@
+    password_confirm: str
+    nickname: Optional[str] = Field(default=None, min_length=2, max_length=20)
+    profile_image_url: Optional[HttpUrl] = None
+
+    @model_validator(mode="after")
+    def password_match(self):
+        if self.password != self.password_confirm:
+            raise ValueError("패스워드 불일치")
+        return self
+
+    @classmethod
+    def as_form(
```

Ruff (0.14.11)

Missing return type annotation for classmethod `as_form`

(ANN206)